vignettes/InstantaneousTimeStep.Rnw

```diff
@@ -42,7 +49,7 @@ names(BadUV) <- sub("_Inst", "", names(BadUV))
 BadData <- mergeNearest(BadQW, "dateTime", right=BadUV, dates.right="dateTime",
   max.diff="4 hours")
 # Rename the left-hand dateTime column
-names(BadData)[9] <- "dateTime"
+names(BadData)[7] <- "dateTime"
```

Comment:
what do you think about
```r
names(BadData)[which(names(BadData)=='dateTime.left')] <- "dateTime"
```
so we're even less likely to rename the wrong column if things change again?